tools/bwa.wdl

```diff
@@ -209,6 +209,10 @@ task bwa_mem {
             description: "Read group information for BWA to insert into the header. BWA format: '@RG\tID:foo\tSM:bar'",
             group: "common",
         }
+        skip_mate_rescue: "Skip mate rescue"
+        skip_pairing: "Skip pairing; mate rescue performed unless `skip_mate_rescue` also in use"
+        split_smallest: "For split alignment, take the alignment with the smallest coordinate as primary"
```

Comment:
What's the alternative?

Reply:
Here is the help text. 
```
       -S            skip mate rescue
       -P            skip pairing; mate rescue performed unless -S also in use
       -5            for split alignment, take the alignment with the smallest coordinate as primary
```
`-P` is the only option ith an entry in the manual:
```
-P | In the paired-end mode, perform SW to rescue missing hits only but do not try to find hits that fit a proper pair.
```

Nothing overly helpful

Reply:
@a-frantz - I've added some additional text, but it relies on google searching and reading Q&A on online forums. I don't see anything definitive from the `bwa` authors.

Reply:
The text for `skip_[mate_rescue,pairing]` looks good now! But I still have my original Q: What's the alternative to "take the alignment with the smallest coordinate as primary"?

Reply:
I assume it's random, since these have the same score (I think).